### ISSUE TITLE
Ubuntu: jv_get_kind(args) == JV_KIND_ARRAY assertion failed.

### ISSUE THREAD

**Issue by vielmetti** [jzelinskie/faq#37]
For the full build log see https://cloud.drone.io/vielmetti/faq/12/1/2 based on the Drone Cloud configuration at https://github.com/vielmetti/faq/blob/vielmetti-drone/.drone.yml

```
=== RUN TestRunFaq/null_flag_input_simple_program
--
109 | faq.test: execute.c:1071: jq_compile_args: Assertion `jv_get_kind(args) == JV_KIND_ARRAY' failed.
```

I suspect that this is because I'm doing a Ubuntu-based build (using the "golang" Docker image as a base) and because jq in that image is 1.5 version.

**Comment by jzelinskie** [comment]
Yeah, in jq 1.6+ args can be arrays or objects. We treat our args as objects, so faq requires jq 1.6+ now.

**Comment by jzelinskie** [comment]
I solved this problem on x86 by building my own base image btw. Alpine edge is the only version of Alpine that has a non-broken libjq 1.6: https://bugs.alpinelinux.org/issues/9503

**Comment by pdecat** [comment]
Work-around on Ubuntu until 1.6 is available there: installing the `jq`, `libjq1` and `libjq-dev` debian sid packages:

* https://packages.debian.org/sid/jq
* https://packages.debian.org/sid/libjq1
* https://packages.debian.org/sid/libjq-dev

Note: only tested on Ubuntu Eoan.

**Comment by jzelinskie** [comment]
The binary releases are also static, so they can be downloaded and executed anywhere.

**Comment by pdecat** [comment]
It's just that I needed to produce a custom build.
